### ISSUE TITLE
Update About Page

### ISSUE THREAD

**Issue by MOQN** [ml5js/ml5-website-v02-gatsby#11]
@shiffman, I apologize for the last minute request. Is there someone suitable/available to update the text on the About page ([webpage](https://ml5-next-gen.netlify.app/about/), [code](https://github.com/ml5js/ml5-website-v02-gatsby/blob/main/src/pages/about.js))? It would be great to have the acknowledgments updated before the launch. Could you please assist with this?

cc'ing @QuinnHe.

**Comment by shiffman** [comment]
Yes, I'd be glad to help with this! @ziyuan-linn has been managing our "all contributors" list on the repo and it would be great to keep these aligned and linked! @sharellb I know you are brand new to the project but perhaps this is something that you can work on with us as you are getting to know all the parts of the project! 

I can start a PR and we can collaborate!
